extensions/postgres/schema/sample_tables.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE sample_user_table(
   user_id VARCHAR(255) NOT NULL,
   first_name VARCHAR(255),
   last_name VARCHAR(255),
+  email VARCHAR(255),
```

Comment:
this is to support the signup demo: https://github.com/xdblab/xdb-golang-samples/blob/main/processes/signup/process.go#L48